slirp4netns.1.md

```diff
@@ -102,6 +105,39 @@ e.g.
 $ sudo sh -c "echo 0   2147483647  > /proc/sys/net/ipv4/ping_group_range"
 ```
 
+# API SOCKET
+
+slirp4netns can provide QMP-like API server over an UNIX socket file:
+
+```console
+$ slirp4netns --api-socket /tmp/slirp4netns.sock ...
+```
+
+**add_hostfwd**: Expose a port
+
+```console
+$ json='{"execute": "add_hostfwd", "arguments": {"proto": "tcp", "host_addr": "0.0.0.0", "host_port": 8080, "guest_addr": "10.0.2.100", "guest_port": 80}}'
+$ echo -n $json | nc -U /tmp/slirp4netns.sock
+{ "return": {}}
```

Comment:
On second thought add-hostfwd should return an integer id and remove-hostfwd should take that id